pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -1529,10 +1529,12 @@ class ClassTypeHintMatch:
                 TypeHintMatch(
                     function_name="speed_count",
                     return_type="int",
+                    mandatory=True,
                 ),
                 TypeHintMatch(
                     function_name="percentage_step",
                     return_type="float",
+                    mandatory=True,
                 ),
                 TypeHintMatch(
```

Comment:
why not also mark this one and the 3 other bellow?

Reply:
They return types have multiple possibilities... so I'm going through first on the ones that have only a single return type